Description: Autotools modifies these files.
A workaround patch needed to enable build twice in a row. There must be a more
suitable way to handle modified files.

In other words, TODO: remove this patch.

--- a/doc/stamp-vti
+++ b/doc/stamp-vti
@@ -1,4 +1,4 @@
-@set UPDATED 29 October 2014
+@set UPDATED 05 November 2014
 @set UPDATED-MONTH October 2014
 @set EDITION 2.20.90-a07a4
 @set VERSION 2.20.90-a07a4
--- a/doc/version.texi
+++ b/doc/version.texi
@@ -1,4 +1,4 @@
-@set UPDATED 29 October 2014
+@set UPDATED 05 November 2014
 @set UPDATED-MONTH October 2014
 @set EDITION 2.20.90-a07a4
 @set VERSION 2.20.90-a07a4
